src/plugins/AriaTablist.js

```diff
@@ -0,0 +1,342 @@
+/* eslint-disable no-underscore-dangle */
+import fastdom from 'fastdom';
+import Aria from './Aria';
+
+/**
+ * Manage tablist elements
+ *
+ * @param {Object} config - Config options for AriaTablist
+ * @param {HTMLElement} config.tablist - The tab elements' parent element
+ * @param {NodeList} config.panels - A list of panel elements; each panel should have a corresponding tab element in config.tabs
+ * @param {MediaQueryList} config.breakpoint - An optional breakpoint definition to be used to init and/or destroy the tablist
```

Comment:
I just added this the other day, but now I'm wondering if it makes more sense to keep this logic in component 🤔